activity/ws/metadata.go

```diff
@@ -1,25 +1,47 @@
 package ws
 
+import "github.com/project-flogo/core/data/coerce"
+
 // Settings are the settings for the websocket proxy
 type Settings struct {
-	URI string `md:"uri,required"`
+	URI           string `md:"uri,required"`
+	AllowInsecure bool   `md:"allowInsecure"`
+	CaCert        string `md:"caCert"`
 }
 
 // Input is the input into the websocket proxy
 type Input struct {
-	Message interface{} `md:"message"`
+	Message     interface{}            `md:"message,required"`
+	PathParams  map[string]string      `md:"pathParams"`
+	QueryParams map[string]interface{} `md:"queryParams"`
+	Headers     map[string]interface{} `md:"headers"`
```

Comment:
As we have params type for Query and Headers, so let's correct Go type tp map[stsring]string

Reply:
done